dac_calibration_server.py

```diff
@@ -259,11 +251,42 @@ def dac(self, c, dac):
         c['DAC'] = dac
         return dac
 
+    @setting(30,
+        'Correct IQ',
+        data=['*(v, v): I/Q data', '*c: I/Q data'],
+        zero_ends='b',
+        returns=['(*i, *i): Dual channel DAC values'])
```

Comment:
This is so much nicer to have a single well-defined return value. Makes it conceivable to move to a more static type system or IDL...